### DISCUSSION TITLE
Issue with typescript

### DISCUSSION THREAD

**Discussion by NarHakobyan** [vuejs/pinia#2540] [Help and Questions]
### Steps to reproduce the bug

Hey, just noticed that storeToRefs returns incorrect types when using ref's inside the store. it wraps 2 Refs

<img width="614" alt="image" src="https://github.com/vuejs/pinia/assets/1781506/015ce5bf-2ea6-4f0a-bbae-d8b899def546">

### Expected behavior

it should not wrap ref 2 times

### Actual behavior

it wraps twice

**Comment by loonyzhang** [comment]
me too, Is there a way to solve it?

**Reply by nf-ssc** [reply]
I was able to fix it by deleting the yarn.lock file and then reinstalling (seems like some dependency issue).

**Comment by NarHakobyan** [comment]
@loonyzhang 

The solution is to verify the Vue versions in your dependencies. For example, if pinia depends on Vue 3.2 but you have installed Vue 3.3, it will result in the installation of two versions of Vue. As a result, the Ref type will be duplicated, leading to the occurrence of this issue.

I Think it's better to move "vue" from dependencies to peer in Pinia

**Reply by loonyzhang** [reply]
thx,  it's good
